docs/src/HowToGuides/Atmos/Microphysics.md

```diff
@@ -145,232 +320,273 @@ where:
+
+There are two additional assuptions that we make to integrate eq.(\ref{eq:accr_sr1}):
+- ``\left|v_{term}(r_i) - v_{term}(r_j)\right| \approx \left| v_{ti} - v_{tj} \right|``
+  We approximate the teminal velocity difference for each particle pair with
+  a difference between mass-weighted mean terminal velocityes and move it outside of the integral.
+  See the discussion in [Ikawa\_and\_Saito\_1991](https://www.mri-jma.go.jp/Publish/Technical/DATA/VOL_28/28_005.pdf) page 88.
+
+-  We assume that ``a(r_i, r_j) = \pi (r_i + r_j)^2``.
+   This corresponds to a geometric formulation of the collision kernel, aka cylindrical fomulation,
+   see [Wang\_et\_al\_2005](https://journals.ametsoc.org/doi/pdf/10.1175/JAS3655.1) for discussion.
+
+
+The eq.(\ref{eq:accr_sr1}) can then be integrated as:
 ```math
-N_{Sc} = \frac{\nu_{air}}{D_{vapor}}
+\begin{align}
+\left. \frac{d \, q_i}{dt} \right|_{accr} & =
+    \pi \, n_0^{i} \, n_0^{j} \, \alpha_j \, E_{ij} \left| v_{ti} - v_{tj} \right|
+    \int_0^\infty \int_0^\infty
+    (r_i + r_j)^2
+    r_{j}^{\beta_j} \,
+    exp(- \lambda_j r_j) \,
+    exp(- \lambda_i r_i) \,
+    dr_i dr_j \\
+    & =
+    \pi \, n_0^{i} \, n_0^{j} \, \alpha_j \, E_{ij} \left| v_{ti} - v_{tj} \right|
+    \left(
+        \frac{2 \Gamma(\beta_j + 1)}{\lambda_i^3 \lambda_j^{\beta_j+1}}
+        + \frac{2 \Gamma(\beta_j + 2)}{ \lambda_i^2 \lambda_j^{\beta_j+2}}
+        + \frac{\Gamma(\beta_j + 3)}{\lambda_i \lambda_j^{\beta_j + 3}}
+    \right)
+\end{align}
 ```
-where:
- - ``\nu_{air}`` is the kinematic viscosity of air.
-The Reynolds number of a rain drop is defined as:
+
+!!! note
+    Both of the assumptions needed to integrate the snow-rain accretion rate could be revisited:
+
+    The discussion on page 88 in
+    [Ikawa\_and\_Saito\_1991](https://www.mri-jma.go.jp/Publish/Technical/DATA/VOL_28/28_005.pdf)
+    suggests an alternative approximation of the velocity difference.
+
+    The ``(r_i + r_j)^2`` assumption for the crossection is inconsistent with the
+    snow crossection used when modelling collisions with cloud water and cloud ice.
+
+
+### Rain evaporation and snow sublimation
+
+We start from a similar equation as when computing snow autoconversion rate
+but integrate it from ``0`` to ``\infty``.
+
+```math
+\begin{equation}
+  \left. \frac{dq}{dt} \right|_{evap\_subl} = \frac{1}{\rho} \int_{0}^{\infty} \frac{dm(r)}{dt} n(r) dr
+\end{equation}
+```
+In contrast to eq.(\ref{eq:mass_rate}), now we are taking into account ventilation effects:
+
 ```math
-N_{Re} = \frac{2 \, r \, v_{drop}(r, \rho)}{\nu_{air}} = \frac{2 v_c(\rho) \, g^{1/2} \, r^{3/2}}{\nu_{air}}
+\begin{equation}
+  \frac{dm}{dt} = 4 \pi \, r \, (S - 1) \, G(T) \, F(r)
+\label{eq:mass_rate2}
+\end{equation}
 ```
+where:
+ - ``F(r)`` is the rain drop ventilation factor defined in (\ref{eq:ventil_factor})
+ - supersaturation S is computed over water or ice
+ - ``L`` is the latent heat of vaporization or sublimation.
```

Comment:
I don't see where `L` pops up in these equations, so do you need to define it here?

Reply:
Thanks. Its a leftover from the times I did not have a separate function for G. Corrected.